### PR TITLE
Update `error-stack` changelog 

### PR DESCRIPTION
**Pull request by indietyp**
## 🌟 What is the purpose of this PR?

I forgot to adjust the changelog in my PRs, which remedies this.

## ⚠️ Known issues

Input on the wording is much appreciated, and things I might've forgotten.

### CODE REVIEW

**Comment by indietyp** [comment]
@Alfred-Mountfield @TimDiekmann mind to look over this?

File: packages/libs/error-stack/CHANGELOG.md [hunk 1]
```diff
@@ -17,10 +17,16 @@ All notable changes to `error-stack` will be documented in this file.
...
+- Add support for related errors and multiple error sources ([#747](https://github.com/hashintel/hash/pull/747))
+- New output for [Debug](https://doc.rust-lang.org/nightly/core/fmt/trait.Debug.html) ([#794](https://github.com/hashintel/hash/pull/794))
+- New hook interface for [Debug](https://doc.rust-lang.org/nightly/core/fmt/trait.Debug.html) ([#794](https://github.com/hashintel/hash/pull/794))
+- `Report::set_debug_hook`, `Report::set_display_hook` no longer return an error ([#794](https://github.com/hashintel/hash/pull/794))
 
 ### Deprecations
 
 - `IntoReport::report`: Use `IntoReport::into_report` instead ([#698](https://github.com/hashintel/hash/pull/698))
+- `Report::set_debug_hook`: Use `Report::install_debug_hook` instead ([#794](https://github.com/hashintel/hash/pull/794))
+- `Report::set_display_hook`([#794](https://github.com/hashintel/hash/pull/794))
```
Comment (TimDiekmann) [inline packages/libs/error-stack/CHANGELOG.md:29]: In #747 we also deprecated
- `Frame::source`in favor of `Frame::sources`
- `Frame::source_mut`in favor of `Frame:: source_mut `
- `Report::backtrace` and `Report::span_trace` in favor of generic access

Comment (indietyp) [reply packages/libs/error-stack/CHANGELOG.md:35]: fixed in [2436d6c](https://github.com/hashintel/hash/pull/973/commits/2436d6cb8b033a8d046b7f19283f69640b582fba)

**Comment by indietyp** [comment]
Should we also include the new methods? Like `Report::current_frames`?

**Comment by TimDiekmann** [comment]
I don't think so as these would be numerous.

File: packages/libs/error-stack/CHANGELOG.md [hunk 2]
```diff
@@ -17,10 +17,22 @@ All notable changes to `error-stack` will be documented in this file.
 - Add support for `Iterator`s ([#716](https://github.com/hashintel/hash/pull/716))
 - Add compatibility for `anyhow` and `eyre` to convert their types into `Report` ([#763](https://github.com/hashintel/hash/pull/763))
 - Add support for [`Error::provide()`](https://doc.rust-lang.org/nightly/std/error/trait.Error.html#method.provide) ([#904](https://github.com/hashintel/hash/pull/904))
+- Add support for related errors and multiple error sources ([#747](https://github.com/hashintel/hash/pull/747))
+- New output for [Debug](https://doc.rust-lang.org/nightly/core/fmt/trait.Debug.html) ([#794](https://github.com/hashintel/hash/pull/794))
+- New hook interface for [Debug](https://doc.rust-lang.org/nightly/core/fmt/trait.Debug.html) ([#794](https://github.com/hashintel/hash/pull/794))
+- `Report::set_debug_hook`, `Report::set_display_hook` no longer return an error ([#794](https://github.com/hashintel/hash/pull/794))
```
Comment (TimDiekmann) [inline packages/libs/error-stack/CHANGELOG.md:23]:
```suggestion
- `Report::set_debug_hook` and `Report::set_display_hook` no longer return an error ([#794](https://github.com/hashintel/hash/pull/794))
```

**Review by TimDiekmann — APPROVED** [review]
Thanks for taking care of this!

**Pull request merged**